x-pack/plugins/security_solution/kibana.json

```diff
@@ -50,6 +51,7 @@
     "kibanaReact",
     "usageCollection",
     "lists",
-    "ml"
+    "ml",
+    "unifiedSearch"
```

Comment:
- [ ] plugin cannot be required and optional the same time. Please keep it only in `requiredPlugins`